src/plugins/intel_cpu/tests/unit/transformations/markup_rope_inputs.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <gtest/gtest.h>
+#include "openvino/opsets/opset1.hpp"
+#include "ov_ops/rotary_positional_embeddings.hpp"
+#include "openvino/pass/manager.hpp"
+#include "transformations/cpu_opset/common/pass/markup_rope_inputs.hpp"
+#include "transformations/rt_info/disable_fp16_compression.hpp"
+
+TEST(TransformationTests, MarkUpRopeInputsTest) {
```

Comment:
Could you please use TransformationTestsF class and compare 2 ov::Models (actual and reference)? You will need to enable RUNTIME_KEYS comparison in order to check runtime info. [Example](https://github.com/openvinotoolkit/openvino/blob/master/src/common/low_precision_transformations/tests/mark_dequantization_subgraph_transformation.cpp)